gui.py

```diff
@@ -778,10 +778,13 @@ def __init__(self, parent, title, url, **kwargs):
         # Creating main gui window
         vbox = wx.BoxSizer(wx.VERTICAL)
         self.toolbar = MainMenuToolBar(self, main_class=self)
-        self.browser_window = chromectrl.ChromeCtrl(self, useTimer=False, url=str(url), hasNavBar=False)
+
+        browser_window  = chromectrl.ChromeCtrl(self, useTimer=False, url=str(url), hasNavBar=False)
+        if hasattr(browser_window, 'stub') != True:
```

Comment:
It is all around wrong, you are creating 2 chrome windows, and will be an exception if it is stub.